modules/ROOT/pages/transaction-management.adoc

```diff
@@ -4,69 +4,100 @@ include::_attributes.adoc[]
+Mule supports Single Resource (Local, the default) and Extended Architecture (`XA`) transaction types (`transactionType`). The only components that can define the transaction type are message sources (For example, `jms:listener` and `vm:listener`) and the Try scope.
+
+The following table describes characteristics of each transaction type and the requisites for an operation to join the transaction:
+
+[%header,cols="20a,35a,45a"]
+|===
+| Transaction Type | Characteristics | Requisites to Join the transaction
+| Single Resource (local)
+|
+* Performs better than XA transactions.
+|
+* The operation must support transactions.
+* All operations inside the transaction must belong to the same Connector. (For example `jms:listener`, `jms:consume` and `jms:publish`).
+* All operations must use the same Global Configuration (`config-ref`).
+| XA
+|
+* Involves using a two-phase commit protocol (2PC).
+* Slower but more reliable than Single Resource transactions.
+|
+* The operation must support transactions.
+
+|===
 
-[source,xml,linenums]
-----
-include::{examplesdir}/transaction-management_1.xml[]
-----
+== Transactional Actions
+
+A Transactional Action (`transactionalAction`) defines the type of action that operations take regarding transactions. The following is a list of all available transactional actions:
+
+* `ALWAYS_BEGIN`: Always start a new transaction when receiving a message. An exception will be thrown if a transaction already exists. Listeners that support transactions have this option.
+
+* `ALWAYS_JOIN`: Always expect a transaction to be in progress when a message is received. If there is no transaction, an exception is thrown. Operations that support transactions have this option.
+
+* `BEGIN_OR_JOIN`: If a transaction is already in progress when a message is received, join the transaction if possible. Otherwise, start a new transaction. Scopes that support transactions provide this setting.
+
+* `JOIN_IF_POSSIBLE`: Join the current transaction if one is available. Otherwise, no transaction is created. Operations that support transactions provide this setting.
+
+* `INDIFFERENT`: Do not treat actions as a transaction. Scopes that support transactions provide this setting.
+
+* `NONE`: Never participate in a transaction. Listeners that support transactions provide this setting.
 
-It's also possible to start a transaction from a message source. In that case, the entire flow is a transaction. This is useful when working with messaging connectors to prevent the consumption of the message if a problem occurs when processing it so you can later retry (because of the rollback).
+* `NOT_SUPPORTED`: Execute outside any existent transaction. Operations that support transactions provide this setting. Scopes that support transactions provide this setting.
```

Comment:
Operations and scopes that support transactions provide this setting. (reduce repetitive words)